packages/x-charts/src/types.ts

```diff
@@ -0,0 +1,8 @@
+import type { MakeOptional } from '@mui/x-internals/types';
+import { ChartsRadiusAxisProps, ChartsRotationAxisProps } from './models/axis';
+import type { AxisConfig, ChartsXAxisProps, ChartsYAxisProps, ScaleName } from './models';
+
+export type XAxis = MakeOptional<AxisConfig<ScaleName, any, ChartsXAxisProps>, 'id'>;
+export type YAxis = MakeOptional<AxisConfig<ScaleName, any, ChartsYAxisProps>, 'id'>;
+export type RotationAxis = AxisConfig<ScaleName, any, ChartsRotationAxisProps>;
+export type RadiusAxis = AxisConfig<'linear', any, ChartsRadiusAxisProps>;
```

Comment:
Looking at it, these should be in `./models/axis` and re-exported at the index. Since we store a lot of our types in the models.

Reply:
Good point! Done